### PR TITLE
Add Marker helper class to iOS SDK

### PR DESCRIPTION
**Pull request by karimnaaji**
- Add `TGMarker` class and update interface to not use marker identifiers and simplify interface.
- Add missing `drawOrder` from interface.
- Add private/internal interface with framework only visibility for `TGMapViewController`.
- Add `markers` accessor to retrieve all marker objects added to a `TGMapViewController`.

### CODE REVIEW

File: platforms/ios/src/TangramMap/TGGeoPolygon.h [hunk 1]
```diff
@@ -83,6 +83,8 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)removeAll;
 
+- (instancetype)userData;
```
Comment (karimnaaji) [inline platforms/ios/src/TangramMap/TGGeoPolygon.h]: This is not used, will remove.

File: platforms/ios/src/TangramMap/TGMapViewController+Internal.h [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+NS_ASSUME_NONNULL_BEGIN
+
+- (void)addMarker:(TGMarker *)marker withIdentifier:(Tangram::MarkerID)identifier;
+
+- (void)removeMarker:(Tangram::MarkerID)identifier;
+
+NS_ASSUME_NONNULL_END
+
+- (TGMarker *)getMarker:(Tangram::MarkerID)identifier;
```
Comment (karimnaaji) [inline platforms/ios/src/TangramMap/TGMapViewController+Internal.h]: Might be a leftover, will remove.

**Comment by tallytalwar** [comment]
@msmollin wanna give this a quick look to check for ios specific things I might be missing. Thanks

**Comment by tallytalwar** [comment]
Circle is actually good, but not updating the status on github. Merging.

**Pull request merged**